randovania/games/dread/logic_database/Artaria.txt

```diff
@@ -1484,8 +1489,12 @@ Extra - asset_id: collision_camera_013
   * Extra - actor_def: actordef:actors/items/item_missiletank/charclasses/item_missiletank.bmsad
   > Door to Intro Room
       Lay Power Bomb
-  > Tunnel to Cold Introduction
-      Lay Any Bomb
+  > Event - Tunnel Speed Booster Blocks (Right)
+      All of the following:
+          Ballspark and Lay Power Bomb
+          Any of the following:
+              Speed Booster Upgrade
+              Charge Beam and Disabled Door Lock Randomizer
```

Comment:
I'm entirerly convinced that the intention for this connection has been that you can also charge speed in the energy recharge room, effectively leading to a situation where, if you can get into this room, you could do the ballspark. Granted, I think it would make sense to add a speed booster conservation trick regardless.

Also, you probably want to have a connection from the pickup to the tunnel that requires having the event. Because, if you break the speed blocks from the left, then it would be easier to back.

Reply:
yeah, you are guaranteed to be able to charge speed if you can get into this room, so the extra reqs are not necessary.

Reply:
However, I disagree with requiring conservation. it removes the logic of breaking the speed block from the other side, then being able to come back through (only applicable in door lock rando)

Reply:
the intention is that without DLR, you can charge speed in energy recharge room, go through the missile door and ballspark. if DLR is enabled you can charge in the bottom of the charge access room with 1+ upgrades. though now that I think about it again it's tight enough that you need 2 if there are shields down there.

Reply:
The intention is that you can charge speed no matter what if you're in this room, since it requires either coming from above, coming from intro room, or just using the speed block tunnel which at that point charging the speed is useless.